### PR TITLE
Don't raise exception

### PR DESCRIPTION
**Pull request by Cory-Christensen**

### CODE REVIEW

**Comment by bfcoder** [comment]
You should probably write a description as to why this is necessary.

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@ class ApplicationController < ActionController::Base
 
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
-  protect_from_forgery with: :exception
+  protect_from_forgery with: :null_session
```
Comment (jbasdf) [inline app/controllers/application_controller.rb]: Why change this to not raise an exception?

Comment (Cory-Christensen) [inline app/controllers/application_controller.rb]: It was breaking something a couple weeks ago...You explained to me that it needed this change....
I haven't run into the issue lately so it's entirely possible this isn't needed anymore.

**Comment by bfcoder** [comment]
@Cory-Christensen Shall we close this PR then?

**Comment by Cory-Christensen** [comment]
Let's close it for now

**Pull request closed without merging**